apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/actions.ts

```diff
@@ -1,12 +1,28 @@
 "use server";
 
 import { TSurvey } from "@formbricks/types/v1/surveys";
-import { deleteSurvey, updateSurvey } from "@formbricks/lib/services/survey";
+import { deleteSurvey, updateSurvey } from "@formbricks/lib/survey/service";
+import { canUserAccessSurvey } from "@formbricks/lib/survey/auth";
+import { authOptions } from "@/app/api/auth/[...nextauth]/authOptions";
+import { getServerSession } from "next-auth";
+import { AuthorizationError } from "@formbricks/types/v1/errors";
 
 export async function surveyMutateAction(survey: TSurvey): Promise<TSurvey> {
```

Comment:
can we rename this to `updateSurveyAction`
(sorry I know it's not your code, but just saw this)